RecoEgamma/EgammaTools/python/slimmedEgammaFromMultiCl_cff.py

```diff
@@ -0,0 +1,134 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoEgamma.EgammaTools.hgcalElectronIDValueMap_cfi import hgcalElectronIDValueMap
+
+hgcElectronID = hgcalElectronIDValueMap.clone()
+patElectronsFromMultiCl = cms.EDProducer("PATElectronProducer",
+    electronSource = cms.InputTag("ecalDrivenGsfElectronsFromMultiCl"),
+    beamLineSrc    = cms.InputTag("offlineBeamSpot"),
+    pvSrc          = cms.InputTag("offlinePrimaryVertices"),
+    addElectronID  = cms.bool(False),
+    addGenMatch    = cms.bool(False),
+    addMVAVariables= cms.bool(False),
+    embedGenMatch              = cms.bool(False),
+    embedGsfElectronCore       = cms.bool(True),
+    embedGsfTrack              = cms.bool(True),
+    embedSuperCluster          = cms.bool(True),
+    embedPflowSuperCluster     = cms.bool(False),
+    embedSeedCluster           = cms.bool(True),
+    embedBasicClusters         = cms.bool(False),
+    embedPreshowerClusters     = cms.bool(False),
+    embedPflowBasicClusters    = cms.bool(False),
+    embedPflowPreshowerClusters= cms.bool(False),
+    embedPFCandidate           = cms.bool(False),
+    embedTrack                 = cms.bool(True),
+    embedRecHits               = cms.bool(False),
+    embedHighLevelSelection    = cms.bool(True),
+    userData = cms.PSet(
+        userClasses = cms.PSet( src = cms.VInputTag('')),
+        userFloats = cms.PSet( src = cms.VInputTag(
+            [cms.InputTag("hgcElectronID", key) for key in hgcElectronID.variables]
+        )),
+        userInts = cms.PSet( src = cms.VInputTag('')),
+        userCands = cms.PSet( src = cms.VInputTag('')),
+        userFunctions = cms.vstring(),
+        userFunctionLabels = cms.vstring()
+    ),
+)
+selectedPatElectronsFromMultiCl = cms.EDFilter("PATElectronSelector",
+    src = cms.InputTag("patElectronsFromMultiCl"),
+    cut = cms.string("!isEB && pt > 15"),
+)
+slimmedElectronsFromMultiCl = cms.EDProducer("PATElectronSlimmer",
+    src = cms.InputTag("selectedPatElectronsFromMultiCl"),
```

Comment:
here and for the patElectrons,
please clone from the non-MultiCl with appropriate modifications.
This is easier to maintain than large copy-pastes of the full definitions

Reply:
Earlier I used that, but the standard miniaod_tools rewrites so much of `patElectrons`, that by the time the modifier runs, I have to remove so many things (which would crash this) that might as well go this direction.

Reply:
please elaborate.
It sounds like what you tried was not done in order.

Reply:
Well, I guess since the era modifier runs after everything else is set up, and since a lot of files end up altering `process.patElectrons` adding a bunch of valuemaps and other things that don't exist for this collection, then I have to remove all those, and the number of config lines ends up being about the same.  Also the default definition is somewhat altered from this one already.

Reply:
I sort of follow your argument for the patElectrons (the miniAOD_tools is modifying it heavily), even though I'd still prefer here to clone from the default to improve maintainability.
The slimmedElectrons (the initial point for the comment) is not modified extensively by miniAOD_tools and cloning it will reduce the number of lines as well as improve maintainability.